{"name": "test_search_space", "status": "failed", "statusDetails": {"message": "AssertionError: Ожидался статус-код 422, но получен 401\nassert 401 == 422\n +  where 401 = <Response [401]>.status_code", "trace": "@allure.feature(\"API\")\n    @allure.story(\"Пробел\")\n    @pytest.mark.api_test\n    @pytest.mark.negative_test\n    def test_search_space():\n        headers = {\n            'content-type': 'application/json',\n            'authorization': f'Bearer {TOKEN}'\n        }\n    \n        response = requests.get(f\"{BASE_URL_2}/search/product?phrase=   \", headers=headers)\n>       assert response.status_code == 422, f\"Ожидался статус-код 422, но получен {response.status_code}\"\nE       AssertionError: Ожидался статус-код 422, но получен 401\nE       assert 401 == 422\nE        +  where 401 = <Response [401]>.status_code\n\ntest_api.py:92: AssertionError"}, "start": 1719414554671, "stop": 1719414554772, "uuid": "25fc98f1-e9d2-437e-990c-1ec467686732", "historyId": "68d74eb797febd16d034aabbadffa454", "testCaseId": "68d74eb797febd16d034aabbadffa454", "fullName": "test_api#test_search_space", "labels": [{"name": "feature", "value": "API"}, {"name": "story", "value": "Пробел"}, {"name": "tag", "value": "negative_test"}, {"name": "tag", "value": "api_test"}, {"name": "suite", "value": "test_api"}, {"name": "host", "value": "LAPTOP-DBMEEV82"}, {"name": "thread", "value": "3716-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_api"}]}